src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -14,9 +14,6 @@
     <AssemblyVersion Condition="'$(TargetGroup)'=='net46'">4.0.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageTargetFramework Condition="'$(TargetGroup)'==''">netstandard1.4</PackageTargetFramework>
-    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true' and '$(TargetGroup)' == ''">win7</PackageTargetRuntime>
-    <PackageTargetRuntime Condition="'$(TargetsLinux)' == 'true' and '$(TargetGroup)' == ''">linux</PackageTargetRuntime>
-    <PackageTargetRuntime Condition="'$(TargetsOSX)' == 'true' and '$(TargetGroup)' == ''">osx.10.10</PackageTargetRuntime>
```

Comment:
Where does the version number come from now that you are defaulting this to osx only in dir.props?